index.html

```diff
@@ -181,131 +192,142 @@ <h2 class="work_title">Multi-Post Stories</h2>
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </p>
+          <h3 class="social_title blue">LET’S CONNECT</h3>
+          <ul class="social_links">
+            <li class="social_link">
+              <a href="#"
+                ><img
+                  class="social_icon"
+                  src="./images/twitter.svg"
+                  alt="Twitter"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img
+                  class="social_icon"
+                  src="./images/linkedin-icon.svg"
+                  alt="Linkedin"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img class="social_icon" src="./images/m.svg" alt="M"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img
+                  class="social_icon"
+                  src="./images/github.svg"
+                  alt="Github"
+              /></a>
+            </li>
+            <li class="social_link">
+              <a href="#"
+                ><img class="social_icon" src="./images/doit.svg" alt="Diot"
+              /></a>
+            </li>
+          </ul>
+          <div class="action">
+            <button type="button" class="default_btn">Get my resume</button>
+          </div>
+        </div>
+        <div class="about-framework">
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Languages
+              <img
+                class="down_arrow_icon"
+                src="./images/down-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+            <li class="ul_item">
+              <img src="./images/js-icon.png" alt="Javascript" /><span
+                >Javascript</span
+              >
+            </li>
+            <li class="ul_item">
+              <img src="./images/html-icon.png" alt="HTML" /><span>HTML</span>
+            </li>
+            <li class="ul_item">
+              <img src="./images/css-icon.png" alt="CSS" /><span>CSS</span>
+            </li>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Frameworks
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
+          <ul class="about_ul">
+            <h3 class="sub_title">
+              Skills
+              <img
+                class="arrow_icon"
+                src="./images/right-arrow-icon.png"
+                alt="down"
+              />
+            </h3>
+          </ul>
         </div>
```

Comment:
- [ ] It is required that on the desktop version of your portfolio, the list of languages should be on a row and each language icon and text should be on a column as we have it on the template you chose. Kindly compare the two designs and work your design in line with that of the template you chose.

### Your Design

![your-design-lang](https://user-images.githubusercontent.com/26597432/199472909-7bc222ad-4b7a-40c6-bea7-ee543579aa3f.png)


### Figma Design

![figma-design-lang](https://user-images.githubusercontent.com/26597432/199472951-9f9d5c24-3b93-418d-b7e6-4b34045da136.png)


- [ ] You're expected to produce a close replica of the template you chose. This involves font, color, margin, padding among others. Kindly look into your headline section of your page on the desktop version and work on the margin of the section. Take a look at both designs and work your design along side the template of your choice.

### Your Design

![your-design-headline](https://user-images.githubusercontent.com/26597432/199473574-7c60a11e-ae46-4708-b607-b065470722a4.png)


### Figma Design

![figma-design-headline](https://user-images.githubusercontent.com/26597432/199473616-e57151ac-5d29-40b0-88a3-8310821726ed.png)